packages/devtools_extensions/bin/_build_and_copy.dart

```diff
@@ -0,0 +1,139 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:args/command_runner.dart';
+import 'package:io/io.dart';
+import 'package:path/path.dart' as path;
+
+/// Command that builds a DevTools extension and copies the built output to
+/// the parent package extension location.
+/// 
+/// Example usage:
+/// 
+/// dart pub global activate devtools_extensions;
+/// dart run devtools_extensions build_and_copy \
+///  --source=path/to/your_extension_web_app \
+///  --dest=path/to/your_pub_package/extension/devtools 
+class BuildExtensionCommand extends Command {
+  BuildExtensionCommand() {
+    argParser
+      ..addOption(
```

Comment:
Since we're explicitly checking for `null` below, consider setting `mandatory: true` and removing the checks.